imagej.html

```diff
@@ -252,6 +252,7 @@
 <p class="first">Save or move the .zip archive to the <span class="code-example">ImageJ &gt; plugins</span> folder.</p>
 <p>Extract the archive into the plugins folder to give a folder named &#8220;OMERO.insight-ij-x&#8221; where x is the version number.</p>
 <p>Leave the complete <span class="code-example">OMERO.insight-ij</span> plugin folder with its components in the <span class="code-example">ImageJ &gt; plugins</span> folder.</p>
+<p>If upgrading from a previous version, delete all old versions of the plugin from the plugins folder.</p>
```

Comment:
Should this step be earlier in the list? Or by this point is it obvious which files came from the old and which from the latest so it's easy to target only the former?